homeassistant/const.py

```diff
@@ -118,6 +118,7 @@ class Platform(StrEnum):
 CONF_BEFORE: Final = "before"
 CONF_BELOW: Final = "below"
 CONF_BINARY_SENSORS: Final = "binary_sensors"
+CONF_BRAND: Final = "brand"
```

Comment:
Please don't add constants to the common constants. We only do that after we see that they have multiple users and are actually common.

Reply:
Understood, I will move it into the component directly, sorry about that